src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -129,5 +193,6 @@ void PromptUserForLicenseIfTrialHasExpired()
         License license;
 
         static ILog Logger = LogManager.GetLogger(typeof(LicenseManager));
+        static HttpClient httpClient = new HttpClient();
```

Comment:
If we needed to use the HttpClient in multiple places or even make multiple calls, then yes I can see how it would be valuable to keep a single instance around, but in this case we're using it in exactly one place to make a single call.